libuserhelper.cpp

```diff
@@ -0,0 +1,234 @@
+#include "libuserhelper.h"
```

Comment:
Missing copyright header, applies to libuserhelper.h too.

Reply:
Fixed